charactersheet/charactersheet/templates/character/treasure.tmpl.html

```diff
@@ -154,6 +154,9 @@
     </div><!-- End lg row-->
   </div> <!-- end panel body  -->
   <div class="container-fluid panel-footer">
-     <span class="pull-right">Total:&nbsp;~<span data-bind='text: worth_in_gold'></span><i>&nbsp;GP</i></span>
+     <span class="pull-left">Total:&nbsp;~<span data-bind='text: worth_in_gold'></span><i>&nbsp;GP</i></span>
+      <span class="pull-right">Total weight:&nbsp;
```

Comment:
Every widget uses that name.  If you want it changed, we can discuss a new name and make a new issue.